### PR TITLE
[IT] translation index.md

### PR DESCRIPTION
**Pull request by fguiso**

### CODE REVIEW

File: docs/it/index.md [hunk 1]
```diff
@@ -140,42 +140,133 @@ learning universitario.
     </tr>
 <!-- =============================== WEEK 6 ================================ -->
     <tr>
-      <td rowspan="3" align="center"><a href="{{site.baseurl}}/it/week06/06">⑥</a></td>
-      <td rowspan="2">Lezione</td>
-      <td><a href="{{site.baseurl}}/it/week06/06-1"></a>-</td>
+      <td rowspan="3" align="center"><a href="en/week06/06">⑥</a></td>
+      <td rowspan="2">Lecture</td>
+      <td><a href="en/week06/06-1">Applicazioni di reti convoluzionali</a></td>
```
Comment (Atcold) [inline docs/it/index.md:145]: This one won't work. It's pointing to the `EN` version.

**Review by Atcold — CHANGES_REQUESTED** [review]
A few edits more, when you can.

**Comment by fguiso** [comment]
Sorry about that, should be fixed now.

File: docs/it/index.md [hunk 2]
```diff
@@ -140,19 +140,19 @@ learning universitario.
...
         <a href="https://drive.google.com/open?id=1sdeVBC3nuh5Zkm2sqzdScEicRvLc_v-F">🖥️</a>
         <a href="https://youtu.be/ycbMGyCPzvE">🎥</a>
       </td>
     </tr>
-    <tr><td><a href="en/week06/06-2">RNNs, GRUs, LSTMs, Attenzione, Seq2Seq, e Reti di Memoria</a></td></tr>
+    <tr><td><a href="it/week06/06-2">RNNs, GRUs, LSTMs, Attenzione, Seq2Seq, e Reti di Memoria</a></td></tr>
     <tr>
-      <td rowspan="1">Practicum</td>
-      <td><a href="en/week06/06-3">Architettura delle RNN e modelli LSTM</a></td>
+      <td rowspan="1">Practica</td>
```
Comment (Atcold) [inline docs/it/index.md:154]: Pratica? Non c'è la "c" prima della "t" in italiano 😝

**Review by Atcold — CHANGES_REQUESTED** [review]

File: docs/it/index.md (hunk 2 above)
Comment (fguiso) [reply docs/it/index.md]: 😅 too much translation, now I'm speaking Itanglish

**Review by Atcold — APPROVED** [review]
Hahahaha! Great!

**Pull request merged**